templates/awx/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,149 @@
+version: '2'
+{{- if not .Values.POSTGRES_SERVER }}
+volumes:
+  awx-pgdata1:
+    driver: ${VOLUME_DRIVER}
+{{- end }}
+services:
+{{- if not .Values.MEMCACHED_SERVER }}
+  memcached:
+    image: memcached:1.5.7-alpine
+    stdin_open: true
+    tty: true
+{{- end }}
+{{- if not .Values.POSTGRES_SERVER }}
+  postgresql:
+    image: postgres:9.6.8-alpine
+    environment:
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
+      POSTGRES_USER: ${DATABASE_USERID}
+      POSTGRES_DB: ${DATABASE_NAME}
+      PGDATA: /var/lib/postgresql/data/pgdata
+    stdin_open: true
+    tty: true
+    volumes_from:
+    - postgresql-data
+    labels:
+      io.rancher.sidekicks: postgresql-data
+  postgresql-data:
+    image: busybox
+    stdin_open: true
+    volumes:
+    - awx-pgdata1:/var/lib/postgresql/data/pgdata
+    tty: true
+    labels:
+      io.rancher.container.start_once: 'true'
+{{- end }}
+  awx-web:
+    image: ansible/awx_web:latest
```

Comment:
Use generic image tag, implicit or explicit, is a bad practice. Could you please use concrete image version??
```
-    image: ansible/awx_web:latest
+    image: ansible/awx_web:1.0
```